tests/forms_tests/contrib/storages.py

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
```

Comment:
"coding" is only required if the file has non-ASCII characters (it'll be removed when we drop Python 2 support too).